SPECS/edk2/edk2.spec

```diff
@@ -221,47 +234,46 @@ Open Virtual Machine Firmware (ia32)
 
 %package ovmf-xen
 Summary:        Open Virtual Machine Firmware, Xen build
-License:        BSD-2-Clause-Patent and OpenSSL
+License:        Apache-2.0 AND BSD-2-Clause-Patent AND BSD-4-Clause AND ISC AND LicenseRef-Fedora-Public-Domain
 Provides:       bundled(openssl)
 BuildArch:      noarch
 %description ovmf-xen
 EFI Development Kit II
 Open Virtual Machine Firmware (Xen build)
 
-%if %{with experimental}
```

Comment:
rather than removing the `with experimental` conditionals, consider changing the definition (at the top of the spec) to `bcond_without experimental`